certif/app/models/certification-candidate.js

```diff
@@ -53,6 +53,13 @@ export default class CertificationCandidate extends Model {
     return '-';
   }
 
+  get subscriptionScope() {
+    if (this.subscriptions.some((sub) => sub.isCore)) {
+      return 'CORE';
```

Comment:
Pour utiliser l'objet metier plutot qu'une magic string

```suggestion
      return SUBSCRIPTION_TYPES.CORE;
```